.github/workflows/automated-tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     timeout-minutes: 30
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x, 17.x]
```

Comment:
so once v18 is released we switch to that instead of v17 right?

Reply:
yes, release date 2022-04-19, we can switch everytime because this is only important on `develop`.